win-odbc64/Common/TransportBase.h

```diff
@@ -134,6 +134,8 @@ enum SRVR_API {
 	SRVR_API_SQLFASTEXECDIRECT,					//OK WMS
 	SRVR_API_SQLFASTFETCH_PERF,					//OK WMS
 	SRVR_API_GETSEGMENTS,						//OK WMS
+    SRVR_API_EXTRACTLOB,                        //OK LOB
+    SRVR_API_UPDATELOB,                         //OK LOB

```

Comment:
You may want to check the tab settings in your editor; the indentation seems to frequently be wrong. Best practice is to store spaces instead of tabs.

Reply:
The above use tab while I use 4 spaces instead of tab.

Reply:
My guess is that the original code uses tabs (which is bad) and the tab setting in github assumes 8 spaces per tab. Consider changing your editor to use 8 spaces per tab.